pages/sign-in/kakao/index.page.tsx

```diff
@@ -3,8 +3,16 @@ import { useEffect } from "react";
 
 import { useRouter } from "next/router";
 
-import { instance } from "@/apis/axios";
+import classNames from "classnames/bind";
+import Lottie from "react-lottie-player";
+
+import { basicInstance, instance, setContext } from "@/apis/axios";
 import { PAGE_PATH } from "@/constants/pagePath";
+import loadingLottie from "@/public/images/login-loading.json";
```

Comment:
```suggestion
import loadingLottie from "@/public/images/login-loading.json";
```
이거는 무슨 파일이죠...? json...?

Reply:
lottie 파일이라고 하는건데, 저희 파일 기준으로 gif에 비해 용량이 1/30 수준으로 작고, 확대해도 깨지지 않는다는 장점이 있어서 사용하게 되었습니다!